src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/_params.py

```diff
@@ -246,3 +246,14 @@ def load_arguments_sb(self, _):
     for scope in ['show', 'complete', 'abort']:
         with self.argument_context('servicebus migration {}'.format(scope)) as c:
             c.argument('namespace_name', arg_type=name_type, help='Name of Standard Namespace')
+
+# Region Namespace NetworkRuleSet
+    with self.argument_context('servicebus namespace network-rule') as c:
+        c.argument('subnet', arg_group='virtualnetworkrule', options_list=['--subnet'], help='Name or ID of subnet. If name is supplied, `--vnet-name` must be supplied.')
+        c.argument('ip_mask', arg_group='ipaddressrule', options_list=['--ip-address'], help='IPv4 address or CIDR range.')
+        c.argument('namespace_name', options_list=['--namespace-name'], id_part=None, help='Name of the Namespace')
```

Comment:
The `arg_group` values here should be "Virtual Network Rule" and "IP Address Rule" respectively. Otherwise these will display as:
`virtualnetworkrule Arguments` 
and
`ipaddressrule Arguments`

Reply:
corrected